internal/console/helpers/helpers.go

```diff
@@ -109,18 +129,34 @@ func PrintResult(summary *model.Summary, failedQueries map[string]error) error {
 		fmt.Printf("%s", WordWrap(err.Error(), "\t\t", 5))
 	}
 	fmt.Printf("------------------------------------\n")
-	for _, q := range summary.Queries {
-		fmt.Printf("%s, Severity: %s, Results: %d\n", q.QueryName, q.Severity, len(q.Files))
-
+	for _, q := range sortBySev(summary.Queries) {
+		fmt.Printf("%s, Severity: %s, Results: %d\n", printer.PrintBySev(q.QueryName,
+			string(q.Severity)), printer.PrintBySev(string(q.Severity), string(q.Severity)), len(q.Files))
+		if !printer.minVersion {
+			fmt.Printf("Description: %s\n", q.Description)
+			fmt.Printf("Platform: %s\n\n", q.Platform)
+		}
 		for i := 0; i < len(q.Files); i++ {
-			fmt.Printf("\t%s:%d\n", q.Files[i].FileName, q.Files[i].Line)
+			fmt.Printf("\t%s %s:%s\n", printer.PrintBySev(fmt.Sprintf("[%d]:", i+1), string(q.Severity)),
+				q.Files[i].FileName, printer.Success.Sprint(q.Files[i].Line))
+			if !printer.minVersion {
+				fmt.Println()
+				for idx, lines := range q.Files[i].VulnLines.Lines {
```

Comment:
Shouldn't `lines` be called `line`, since it refers to a single line?

Reply:
resolved